src/components/Modal/ProposalPlugins.vue

```diff
@@ -28,7 +28,7 @@ function showButton(pluginObj) {
   );
   return pluginsWithParams
     .map(plugin => plugin.toLowerCase())
-    .includes(pluginObj.name.toLowerCase());
+    .includes(pluginObj.name.toLowerCase().split(' ')[0]);
```

Comment:
I dont understand why you use name here? This look like an ugly fix, it should use the plugin key, we can't assume the name of a plugin will match the key. We don't need lowercase or anything.

Reply:
There isn't a plugin key as far as I can see.

Reply:
These are the keys https://github.com/snapshot-labs/snapshot.js/blob/master/src/plugins/index.ts#L8-L12

Reply:
This is what the plugin object looks like `{author: "Gnosis", version: "1.0.0", name: "SafeSnap", website: "https://safe.gnosis.io"}`

Reply:
Oh I think I see what I could do.